### DISCUSSION TITLE
[Upgrade] Water Walking

### DISCUSSION THREAD

**Discussion by JohnMetroid** [Malcolm-Q/LC-LateGameUpgrades#531] [General]
would be great to get an upgrade that reduces the move speed penalty in water

**Comment by WhiteSpike** [comment]
3.10.0 release should have this implemented in "Rubber Boots" upgrade.

**Comment by JohnMetroid** [comment]
Excited to check this out soon.  Thanks for all of the work you've put into LGU :)
